controllers/scc.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
```

Comment:
can you try below once?

``` diff
diff --git a/controllers/scc.go b/controllers/scc.go
index a987590..f0a9d62 100644
--- a/controllers/scc.go
+++ b/controllers/scc.go
@@ -45,17 +45,17 @@ func (c openshiftSccs) ensureCreated(r *LVMClusterReconciler, ctx context.Contex
 
                sccs := getAllSCCs(lvmCluster.Namespace)
                for _, scc := range sccs {
-                       found, err := r.SecurityClient.SecurityContextConstraints().Get(scc.Name, metav1.GetOptions{})
+                       found, err := r.SecurityClient.SecurityContextConstraints().Get(ctx, scc.Name, metav1.GetOptions{})
                        if err != nil && errors.IsNotFound(err) {
                                r.Log.Info("Creating SecurityContextConstraint.", "SecurityContextConstraint", scc.Name)
-                               _, err := r.SecurityClient.SecurityContextConstraints().Create(scc)
+                               _, err := r.SecurityClient.SecurityContextConstraints().Create(ctx, scc, metav1.CreateOptions{})
                                if err != nil {
                                        return fmt.Errorf("unable to create SCC %+v: %v", scc, err)
                                }
                        } else if err == nil {
                                scc.ObjectMeta = found.ObjectMeta
                                r.Log.Info("Updating SecurityContextConstraint.", "SecurityContextConstraint", scc.Name)
-                               _, err := r.SecurityClient.SecurityContextConstraints().Update(scc)
+                               _, err := r.SecurityClient.SecurityContextConstraints().Update(ctx, scc, metav1.UpdateOptions{})
                                if err != nil {
                                        r.Log.Error(err, "Unable to update SecurityContextConstraint.", "SecurityContextConstraint", scc.Name)
                                        return fmt.Errorf("unable to update SCC %+v: %v", scc, err)
```

Reply:
That was what I had originally but that did not match the v3.9 signatures

Reply:
It does work with the updated packages .